core/common/src/main/java/alluxio/security/authorization/ModeParser.java

```diff
@@ -120,6 +121,7 @@ private Mode parseSymbolic(String value) {
             break;
           default:
             // Should never get here as already checked for invalid targets
```

Comment:
nit: it bugs me that the comment and exception are in different order between the two default cases :-)